enclave/tls_cert.c

```diff
@@ -260,6 +262,97 @@ oe_result_t oe_get_attestation_certificate_with_evidence(
     return result;
 }
 
+/**
+ * oe_get_attestation_certificate_with_evidence_v2
+ *
+ * Similar to oe_get_attestation_certificate_with_evidence, this function
+ * generates a self-signed x.509 certificate with embedded evidence generated by
+ * an attester plugin for the enclave, but it also allows a user to pass in
+ * optional parameters.
+ *
+ * @experimental
+ *
+ * @param[in] format_id The format id of the evidence to be generated.
+ *
+ * @param[in] subject_name a string containing an X.509 distinguished
+ * name (DN) for customizing the generated certificate. This name is also used
+ * as the issuer name because this is a self-signed certificate
+ * See RFC5280 (https://tools.ietf.org/html/rfc5280) for details
+ * Example value "CN=Open Enclave SDK,O=OESDK TLS,C=US"
+ *
+ * @param[in] private_key a private key used to sign this certificate
+ * @param[in] private_key_size The size of the private_key buffer
+ * @param[in] public_key a public key used as the certificate's subject key
+ * @param[in] public_key_size The size of the public_key buffer.
```

Comment:
Please also be consistent as to whether param descriptions end in a period or not